python/cuml/comm/serialize.py

```diff
@@ -22,10 +22,6 @@
 import cudf.comm.serialize  # noqa: F401
 
 
-def serialize_mat_descriptor(m):
```

Comment:
Oh wow, I completely forgot about this guy. Are we sure the cusparse arrays will properly serialize and be able to be passed around across dask workers without this in there? 

I recall this was quite the but to track down initially, though it was also quite awhile ago so I suspect it could be fixed in cupy by now.